stage1/usr_from_kvm/lkvm/patches/do_synchronous_writes.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/builtin-run.c b/builtin-run.c
+index edcaf3e..f2f2d73 100644
+--- a/builtin-run.c
++++ b/builtin-run.c
+@@ -590,7 +591,7 @@ static struct kvm *kvm_cmd_run_init(int argc, const char **argv)
+ 	}
+ 
+ 	if (kvm->cfg.using_rootfs) {
+-		strcat(real_cmdline, " rw rootflags=trans=virtio,version=9p2000.L,cache=loose rootfstype=9p");
++		strcat(real_cmdline, " rw rootflags=trans=virtio,version=9p2000.L,cache=none rootfstype=9p");
```

Comment:
Argh, was sure that commented this. We had an issue connected with these caches. Without this change some data could be lost if they are saved by application just before pod stopping. For example if I started docker://busybox in interactive mode, added some volume, echoed some value into new file in this volume and then quickly closed pod - file was created, but without expected content inside.
Changing this option helped.
Change was done basing on information from https://www.kernel.org/doc/Documentation/filesystems/9p.txt